jonesy/jobs.py

```diff
@@ -27,36 +27,36 @@ def run(self):
         if self.name == 'upload_advisors':
             self.upload_query_results(
                 queries.get_advisor_notes_access(),
-                f'sis-data/sis-sysadm/{daily_path}/advisors/advisor-note-permissions.gz',
+                f'sis-data/sis-sysadm/{daily_path}/advisors/advisor-note-permissions/advisor-note-permissions.gz',
             )
             self.upload_query_results(
                 queries.get_instructor_advisor_relationships(),
-                f'sis-data/sis-sysadm/{daily_path}/advisors/instructor-advisor-map.gz',
+                f'sis-data/sis-sysadm/{daily_path}/advisors/instructor-advisor-map/instructor-advisor-map.gz',
             )
         elif self.name == 'upload_recent_refresh':
             recency_cutoff = datetime.fromtimestamp(time.time() - (RECENT_REFRESH_CUTOFF_DAYS * 86400))
             for term_id in self.get_current_term_ids():
                 self.upload_query_results(
                     queries.get_recent_instructor_updates(term_id, recency_cutoff),
-                    f'sis-data/{daily_path}/instructor-updates-{term_id}.gz',
+                    f'sis-data/{daily_path}/instructor_updates/instructor-updates-{term_id}.gz',
                 )
                 self.upload_query_results(
                     queries.get_recent_enrollment_updates(term_id, recency_cutoff),
-                    f'sis-data/{daily_path}/enrollment-updates-{term_id}.gz',
+                    f'sis-data/{daily_path}/enrollment_updates/enrollment-updates-{term_id}.gz',
```

Comment:
Sharp eyes will note that we have some inconsistent hyphen-versus-underscore business going on here, but this is what Junction is currently uploading and what Nessie expects, so I don't want to mess with it during cutover.